src/js/components/Paragraph/doc.js

```diff
@@ -29,3 +29,21 @@ export const doc = Paragraph => {
 
   return DocumentedParagraph;
 };
+
+export const themeDoc = {
+  'global.colors.text': {
+    description: 'The text color used inside the Paragraph.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ dark: '#f8f8f8', light: '#444444' }",
+  },
+  'global.text': {
+    description: 'The text size of the Paragraph text.',
+    type: 'string',
+    defaultValue: 'medium',
+  },
+  'paragraph.extend': {
```

Comment:
We also need to include `paragraph.<size>` from base.js, which controls the font-size, line-height, and max-width of the paragraph.

Reply:
Done. Let me know if it is compliant with what you had in mind.